dev/Materials/Acrylic/AcrylicBrush_rs1_themeresources.xaml

```diff
@@ -46,6 +46,10 @@
             <SolidColorBrush x:Key="SystemControlAltHighAcrylicElementBrush" Color="{StaticResource SystemAltHighColor}" />
             <SolidColorBrush x:Key="SystemControlTransientBorderBrush" Color="#000000" Opacity="0.36" />
             <StaticResource x:Key="SystemControlTransientBackgroundBrush" ResourceKey="SystemChromeMediumLowColor" />
+            <StaticResource x:Key="SystemControlTransientAcrylicElementBrush" ResourceKey="SystemChromeMediumLowColor" />
+            <SolidColorBrush x:Key="SystemControlTransientBackgroundInverseBrush" Color="#F2F2F2" />
+            <StaticResource x:Key="SystemControlBaseAcrylicBrush" ResourceKey="SystemChromeMediumLowColor" />
+            <SolidColorBrush x:Key="SystemControlDefaultBrighteningBrush" Color="#FFFFFF" Opacity="0.419"/>
```

Comment:
Maybe we don't need this resource redefined in the RS1 file? It's a SolidColorBrush to begin with and shouldn't have any back compat issues.
(Again, looking to @teaP as the authority here. Just my thought.)

Reply:
I added it to every file as I was not sure how the resource dictionary actually get merged and if it might get dropped if only specified in one dictionary.